docs/quantum_oscilator/architecture.md

```diff
@@ -0,0 +1,81 @@
+# Architecture of Neutal Network
+
+## Parent classes
```

Comment:
Parent class of what? There is no class signature anywhere, please add it

Reply:
Nie rozumiem jest tak samo jak u kulisia, mówiliśmy na spotkaniu we wtorek że parent klas co keras.model

Reply:
Nie rozumiem jest tak samo jak u kulisia, mówiliśmy na spotkaniu we wtorek że parent class to keras.model*

Reply:
Chodzi o to, że nigdzie nie pojawia się sygnatura klasy, tj nagłówek class QONetwork, zobacz na branchu feature/docs-params jak wygląda plik params.md

Reply:
Niżej jest więcej pytań, proszę zerknij